docs/content/getting-started.md

```diff
@@ -242,34 +264,14 @@ query findTodos {
 
 ## Finishing touches
 
-gqlgen is still unstable, and the APIs may change at any time. To prevent changes from ruining your day make sure
-to lock your dependencies:
-
-*Note*: If you dont have dep installed yet, you can get it [here](https://github.com/golang/dep)
-
-First uninstall the global version we grabbed earlier. This is a good way to prevent version mismatch footguns.
-
-```bash
-rm ~/go/bin/gqlgen
-rm -rf ~/go/src/github.com/99designs/gqlgen
-``` 
-
-Next install gorunpkg, its kind of like npx but only searches vendor.
-
-```bash
-dep init
-dep ensure
-```
-
 At the top of our resolvers.go a go generate command was added that looks like this:
 ```go
-//go:generate gorunpkg github.com/99designs/gqlgen
+//go:generate go run ./scripts/gqlgen.go
```

Comment:
did you update the template for resolvers.go?

Reply:
Ah interesting.  Yeah what should we do there?  We can't assume users will have a script to run at `scripts/gqlgen.go`.